config.py

```diff
@@ -0,0 +1,51 @@
+import pydantic
+from typing import Optional, List, Dict, Union, Any
+from aws_cdk import aws_ec2
+
+
+class Config(pydantic.BaseSettings):
+    project_id: Optional[str] = pydantic.Field(
+        description="Project ID", default="cdk-eoapi-demo"
+    )
+    stage: Optional[str] = pydantic.Field(
+        description="Stage of deployment", default="test"
+    )
+    auth_provider_jwks_url: Optional[str] = pydantic.Field(
+        description="Auth Provider JSON Web Key Set URL for ingestion authentication. If not provided, no authentication will be required."
+    )
+    data_access_role_arn: Optional[str] = pydantic.Field(
+        description="Role ARN for data access, if none will be created at runtime.",
+    )
+    db_instance_type: Optional[str] = pydantic.Field(
```

Comment:
Let's confirm with @alukach what we should be considering for minimum size for the instance_type.

Reply:
Sure. The reason I added the instance type and the allocated storage as options that are visible at the top is that I myself by mistake overlooked them when copying the ASDI code, and ended up with too much capacities that cost money even if we don't use them (you pay for the full TiB allocated even if you don't use it for example).

So I believe it's good to have this option clearly visible, but agreed we should talk about the defaults, I am not knowledgeable enough on that, just picked small options.

Reply:
Notes from call : let's go with these small values if that works in tests. But we should talk with David Bitner about documentation regarding the best choices in terms of sizes in production.